crates/bevy_hierarchy/src/hierarchy/hierarchy_maintenance_system.rs

```diff
@@ -78,6 +78,8 @@ mod test {
         world::World,
     };
 
+    use bevy_transform::components::Transform;
```

Comment:
I feel like this should be rectified. `bevy_hierarchy` should be a dependency of `bevy_transform` not the other way around, even for test dependencies.